proposals/29:upgrade-9/test.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+set -e
```

Comment:
docs say...

> "set -euo pipefail" is recommended.

you didn't want to risk compat problems?

Reply:
yeah, but I should follow the advice. updated.

Reply:
I had to revert it to get to green. Something for another time.